### ISSUE TITLE
[BUG] When Docker-Compose Up arizephoenix/phoenix:latest container error  : ModuleNotFoundError: No module named 'psycopg2'

### ISSUE THREAD

**Issue by mhrsavci** [Arize-ai/phoenix#4301]
My docker-compose.yaml looks like : 

```services:
  phoenix:
    image: arizephoenix/phoenix:latest
    depends_on:
      - db
    ports:
      - 6006:6006
      - 4317:4317
    environment:
      - PHOENIX_SQL_DATABASE_URL=postgresql://postgres:postgres@db:5432/postgres
  db:
    image: postgres
    restart: always
    environment:
      - POSTGRES_USER=postgres
      - POSTGRES_PASSWORD=postgres
      - POSTGRES_DB=postgres
    ports:
      - 5432
    volumes:
      - database_data:/var/lib/postgresql/data
volumes:
  database_data:
    driver: local```

**Comment by mikeldking** [comment]
Thanks for reporting this @glmourad @mhrsavci - we will get this fixed. In the meantime maybe pin to an earlier version that works. Apologies for the bug.

**Comment by RogerHYang** [comment]
This is fixed in 4.26.0. Please try again. Thank you.
